test/org/sagebionetworks/bridge/validators/DataGroupsValidatorTest.java

```diff
@@ -0,0 +1,45 @@
+package org.sagebionetworks.bridge.validators;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.sagebionetworks.bridge.exceptions.InvalidEntityException;
+import org.sagebionetworks.bridge.models.accounts.DataGroups;
+
+import com.google.common.collect.Sets;
+
+public class DataGroupsValidatorTest {
```

Comment:
For completeness, how about the following tests:
* a DataGroups that passes validation
* a study with no data groups, but the DataGroups object has data groups
* an empty DataGroup